.eslintrc.js

```diff
@@ -381,6 +381,12 @@ module.exports = {
         'unicorn/prefer-dom-node-text-content': 'off',
       },
     },
+    {
+      files: ['**/vite.config.ts'],
+      rules: {
+        'import/no-default-export': 'off',
+      },
+    },
```

Comment:
```suggestion
```

not needed

Reply:
this is already in your PR, and it does away with meaningless eslint errors

Reply:
this rule is not setup, it's setup in my pr only for monaco-graphql directory, so I don't know where your errors appears